examples/window/window/app.py

```diff
@@ -183,9 +189,17 @@ def startup(self):
                 btn_do_report,
                 btn_change_content,
                 btn_hide,
+                btn_beep,
             ],
             style=Pack(direction=COLUMN),
         )
+        self.scroller = toga.ScrollContainer(
+            horizontal=False,
+            vertical=True,
+            style=Pack(flex=1, padding=10),
```

Comment:
This puts the padding on the *outside* of the scroll container, so there's a border around the scrolling content. We don't need any padding there.

Reply:
ok, removed padding.